Source/BoundaryConditions/PML.cpp

```diff
@@ -565,35 +565,43 @@ PML::PML (const int lev, const BoxArray& grid_ba, const DistributionMapping& gri
     // In order to implement this, a reduced domain is created here (decreased by ncells in all direction)
```

Comment:
Maybe the term "reduced domain" is not appropriate anymore in this comment, since it is not a single contiguous domain anymore.